todo_app/app.py

```diff
@@ -8,4 +9,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items = items)
+    
+@app.route('/add_todo',methods=['POST'] )
+def add_todo():
+    new_todo_title = request.form.get('title')
+    add_item(new_todo_title)  
+    return redirect('/')     
+
+
+
+
+
+    
```

Comment:
Really minor, but some teams will be fussy about spare lines at the bottom of files - no strong reason but it can make reviews longer/harder to read. I wouldn't usually flag it, but there's very little else to comment on!